src/components/categoryitems.main.jsx

```diff
@@ -96,45 +95,39 @@ class CategoryItemsMain extends React.Component {
   render() {
     const { categoryModel, selfUri } = this.state;
     const { categoryUrl } = this.props;
-    if (categoryModel.links.length > 0 && categoryModel._items && categoryModel._items[0].links.length === 0 && selfUri === categoryUrl) {
+    const isLoading = categoryModel.links.length === 0 || selfUri !== categoryUrl;
+
+    if (isLoading) {
       return (
-        <div className="category-items-container container">
-          <div data-region="categoryTitleRegion" style={{ display: 'block' }}>
-            <div>
-              <h1 className="view-title">
-                {categoryModel['display-name']}
-              </h1>
-            </div>
-            <br />
-            <div data-region="categoryTitleRegion" style={{ display: 'block' }}>
-              <div>
-                <h3>
-                  {intl.get('no-products-found')}
-                </h3>
-              </div>
-            </div>
-          </div>
-        </div>
+        <div className="loader" />
       );
     }
-    if (categoryModel.links.length > 0 && selfUri === categoryUrl) {
-      return (
-        <div className="category-items-container container">
-          <div data-region="categoryTitleRegion" style={{ display: 'block' }}>
+
+    const products = categoryModel._items ? categoryModel._items[0] : categoryModel;
+    const noProducts = !products || products.links.length === 0;
+
+    return (
+      <div className="category-items-container container-3">
```

Comment:
I noticed categoryitems.main.jsx *does not* have a .less file? Did you provide the styling in another file for these items?

a {
  color: #333333;
  text-decoration: none;
}

anchor for individual item names is missing this css.

Reply:
Yes, I've styled product cards in `productlistitem.main.less` and styled pagination in pagination component. Basic layout for this page is styled in `.container-3` class in `style.less`. It's a basic three state container that can be used for multiple pages.